src/core/api_client/tests/factories/token.factory.ts

```
@@ -5,28 +5,34 @@ export class TokenFactory {
+      exp: now + 3600, // 1 hour from now
+      iat: now,
       ...payload,
     };
 
     return new SignJWT({ ...defaultPayload })
       .setProtectedHeader({ alg: 'HS256', typ: 'JWT' })
-      .setIssuedAt(defaultPayload.iat || 1700000000)
-      .setExpirationTime(defaultPayload.exp || 9999999999)
+      .setIssuedAt(defaultPayload.iat!)
+      .setExpirationTime(defaultPayload.exp!)
       .sign(this.SECRET);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)